src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -195,3 +198,62 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn reaction-authors
+  [reaction-authors show-reaction-author-list?]
+  (let [selected-tab (reagent/atom (first (keys reaction-authors)))]
+    [:f>
+     (fn []
+       (rn/use-effect (fn []
+                        (fn []
+                          (reset! show-reaction-author-list? false)))
+                      [reaction-authors])
+       [:<>
+        [quo/tabs
+         {:style          style/tabs
+          :size           32
+          :on-change      #(reset! selected-tab %)
+          :default-active @selected-tab
+          :data           (mapv (fn [[reaction-type author-details]]
+                                  ^{:key (str reaction-type)}
+                                  {:id                  reaction-type
+                                   :style               style/tabs
+                                   :label               [rn/view {:style style/tab}
```

Comment:
Now looking in retrospective, I was thinking that since `quo/tabs` uses a `FlatList`, we shouldn't be building hiccup in the `data` prop. There should ideally be a `render-fn` prop exposed by the `tabs` component. In this way, it would also be possible to extract the renderer that here is built inline to a separate var. Anyway, ideas for performance improvements.

Reply:
Refactor in a follow up?

Reply:
It's certainly an interesting refactor, but given other priorities, it's totally up to you @ibrkhalil.